fern/pages/docs/components/code-blocks.mdx

```diff
@@ -361,3 +361,62 @@ System.out.println("Second code block - syncs with the one above!");
 puts "Second code block - syncs with the one above!"
 ```
 </CodeBlocks>
+
+### Override synchronization
+
+You can override the synchronization of code blocks by setting the `for` prop.
+
+<Tabs>
+    <Tab title="Example">
+        <CodeGroup>
+            ```bash title="npm" for="npm"
```

Comment:
is there an example where `title` and `for` aren't the same for every code block? would be more helpful as a viewer trying to understand what's going on